### PR TITLE
Remove windowsContainerAppNotAllowedForPublishProfile validator.

### PR DESCRIPTION
**Pull request by chunye**
I ran "npm run build" after my code change. This resulted in all the changes in JS files.  Let me know if the PR needs to include them.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "webapps-deploy",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (AmrutaKawade) [inline package.json:3]: no need to change this

Comment (AmrutaKawade) [inline package.json:3]: Update version of this library azure-actions-appservice-rest

Comment (chunye) [reply package.json:3]: Will do.

Comment (chunye) [reply package.json]: Thanks.  Changed the version back.

Comment (chunye) [reply package.json]: Fixed now.

**Review by BALAGA-GAYATRI — APPROVED** [review]

**Comment by chunye** [comment]
This PR https://github.com/microsoft/pipelines-appservice-lib/pull/40 has been merged and package published. 

[3:32 AM] Balaga Gayatri
    PR is merged and package got published

**Comment by JasonFreeberg** [comment]
@BALAGA-GAYATRI -- Can we merge this PR now as well? It sounds like this repo was updated to use the latest pipelines-appservice-lib version.

**Pull request merged**